cmd/manager/main.go

```diff
@@ -161,10 +161,11 @@ func serveCRMetrics(cfg *rest.Config) error {
 		return err
 	}
 	// Get the namespace the operator is currently deployed in.
-	operatorNs, err := k8sutil.GetOperatorNamespace()
-	if err != nil {
-		return err
-	}
+	// operatorNs, err := k8sutil.GetOperatorNamespace()
```

Comment:
No commented out code please

Reply:
This is not from this PR this is from the k8s-1.16 one, let me remove this change.